ReleaseTooling/Template/README.md

```diff
@@ -34,17 +34,18 @@ To integrate a Firebase SDK with your app:
    box that appears, make sure the target you want this framework to be added to
    has a checkmark next to it, and that you've selected "Copy items if needed."
 
-   > ⚠ Do not add the Firebase frameworks to the **Embed Frameworks** Xcode build
-   > phase. The Firebase frameworks are not embedded dynamic frameworks, but are
-   > [static frameworks](https://www.raywenderlich.com/65964/create-a-framework-for-ios)
-   > which cannot be embedded into your application's bundle.
+7. If using Xcode 15, embed each framework that was dragged in. Navigate to the
+   target's _General_ settings and find _Frameworks, Libraries, & Embedded
+   Content_. For each framework dragged in from the `Firebase.zip`, select
+   **Embed & Sign**. This step will enable privacy manifests to be picked up by
```

Comment:
> For each framework dragged in from the `Firebase.zip`, select
   **Embed & Sign**.

I'm opting to recommend to embed and sign **each** framework, regardless if it contains a privacy manifest. The alternative way would be to only embed and sign SDKs that have privacy manifests, but encoding that in the zip instructions may add more complexity that it's worth. Thoughts?

Reply:
Agreed, and eventually they may all have manifests anyway.